Mathlib/Algebra/Module/LocalizedModule.lean

```diff
@@ -1037,6 +1037,52 @@ theorem mk'_surjective : Function.Surjective (Function.uncurry <| mk' f : M × S
   obtain ⟨⟨m, s⟩, e : s • x = f m⟩ := IsLocalizedModule.surj S f x
   exact ⟨⟨m, s⟩, mk'_eq_iff.mpr e.symm⟩
 
+section liftOfLE
+
+variable {M₁ M₂} [AddCommMonoid M₁] [AddCommMonoid M₂] [Module R M₁] [Module R M₂]
+variable (S₁ S₂ : Submonoid R) (h : S₁ ≤ S₂) (f₁ : M →ₗ[R] M₁) (f₂ : M →ₗ[R] M₂)
+variable [IsLocalizedModule S₁ f₁] [IsLocalizedModule S₂ f₂]
+
+/-- The natural map `Mₛ →ₗ[R] Mₜ` if `s ≤ t` (in `Submonoid R`). -/
+noncomputable
+def liftOfLE : M₁ →ₗ[R] M₂ :=
+  lift S₁ f₁ f₂ fun x ↦ map_units f₂ ⟨x.1, h x.2⟩
```

Comment:
Can the `(S₁ S₂ : Submonoid R)` be implicit here?

Reply:
I don't think we usually have the inequality floating around so I think we should keep S explicit here.

Reply:
Also fine.